func_adl_xAOD/template/cms/r5/BuildFile.xml

```diff
@@ -0,0 +1,14 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/ParameterSet"/>
+<!-- ******** -->
+<use name="DataFormats/TrackReco"/>
+<use name="CommonTools/UtilAlgos"/>
+<use name="PhysicsTools/UtilAlgos"/>
+<use name="DataFormats/MuonReco"/>
```

Comment:
Should this be part of the XML spec, or somethign that is "included" if muons are referenced? Sort of the way I think it is done on the ATLAS side.

I'm thinking ahead here to new objects we might want to add.

This is fine to be marked as a "TODO" for later circuling back to.

Reply:
Added it to issues: https://github.com/sudo-panda/func_adl_xAOD/issues/2